src/main/app-paths/app-paths-request-channel-listener.injectable.ts

```diff
@@ -3,16 +3,16 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import type { RequestChannelListener } from "../../common/utils/channel/request-channel-listener-injection-token";
-import { requestChannelListenerInjectionToken } from "../../common/utils/channel/request-channel-listener-injection-token";
+import type { RequestChannelHandlerDescriptor } from "../../common/utils/channel/request-channel-listener-injection-token";
+import { requestChannelHandlerInjectionToken } from "../../common/utils/channel/request-channel-listener-injection-token";
 import type { AppPathsChannel } from "../../common/app-paths/app-paths-channel.injectable";
 import appPathsChannelInjectable from "../../common/app-paths/app-paths-channel.injectable";
 import appPathsInjectable from "../../common/app-paths/app-paths.injectable";
 
 const appPathsRequestChannelListenerInjectable = getInjectable({
   id: "app-paths-request-channel-listener",
 
-  instantiate: (di): RequestChannelListener<AppPathsChannel> => {
+  instantiate: (di): RequestChannelHandlerDescriptor<AppPathsChannel> => {
```

Comment:
Nit: The naming is not symmetrical between the two channel types, as there is no `MessageChannelListenerDescriptor` to contrast `RequestChannelHandlerDescriptor`.